sys/fmt/fmt.c

```diff
@@ -53,6 +53,16 @@ static inline int _is_digit(char c)
     return (c >= '0' && c <= '9');
 }
 
+static inline int _is_upper(char c)
```

Comment:
Using the `ctype.h` equivalents here and below instead of these functions adds exactly 400B to the uuid unittest (samr21-xpro board). Which is exactly the reason why the fmt library exists, most of these functions have some equivalent in libc.